.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -33,8 +33,17 @@ Developer considerations
   - If this is a PATCH version, what was fixed?
 - [ ] Update the appropriate documentation. Keep in mind that this mono repository has README files at the repository, project, and component levels. We want developers to be able to find what they need from the main README, so make sure to add links to component documentation if adding something new.
 
+- ** On any task **
```

Comment:
I think the stuff above this can be removed since we don't do version bumps or changelog entries on very many PRs - just the release PRs